{
  "uid" : "ac869156051426e9",
  "name" : "test_upload_data",
  "fullName" : "testproject.test_upload_data.TestUploadData#test_upload_data",
  "historyId" : "807f61ffc93b050ad6f945b881d59179",
  "time" : {
    "start" : 1629047159143,
    "stop" : 1629047159154,
    "duration" : 11
  },
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'upload_data.csv'",
  "statusTrace" : "self = <test_upload_data.TestUploadData object at 0x7f5d46ae36d0>\n\n    def test_upload_data(self):\n        g_feed = self.driver.find_elements_by_class_name(\"article-preview\")\n        feed_upload = 0\n>       with open('upload_data.csv', encoding=\"utf-8\") as ud_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'upload_data.csv'\n\ntestproject/test_upload_data.py:33: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "xunit_setup_method_fixture_TestUploadData",
    "time" : {
      "start" : 1629047149314,
      "stop" : 1629047159142,
      "duration" : 9828
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "xunit_setup_method_fixture_TestUploadData::0",
    "time" : {
      "start" : 1629047159202,
      "stop" : 1629047161366,
      "duration" : 2164
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "testproject"
  }, {
    "name" : "suite",
    "value" : "test_upload_data"
  }, {
    "name" : "subSuite",
    "value" : "TestUploadData"
  }, {
    "name" : "host",
    "value" : "fv-az83-229"
  }, {
    "name" : "thread",
    "value" : "3905-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testproject.test_upload_data"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "754793d696aaa2ad",
        "reportUrl" : "https://nikoteszt.github.io/conduit/41//#testresult/754793d696aaa2ad",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: './upload_data.csv'",
        "time" : {
          "start" : 1629046478930,
          "stop" : 1629046478939,
          "duration" : 9
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "ac869156051426e9.json",
  "parameterValues" : [ ]
}